browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -274,6 +274,8 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			// encode spaces
 			var encodedCommand = data.replace(' ', '%20');
 			builder.map.sendUnoCommand(encodedCommand);
+			// perform post-processing
+			builder.map.fire('jsdialog' + eventType, { uno: data });
```

Comment:
I said it before, this way you make it not working for non-jsdialog.

It was already in a good place (`sendUnoCommand` is central place working for all cases). Only please remove unnecessary "eventType". I was just asking for less code (one param) :)

Why do you need that "eventType"? . This should happen not only "on click" but in general after `.uno:SidebarDeck.PropertyDeck` command is called (doesn't matter how triggered), is that right?

Also dynamically created fire "identifiers" are not a good idea in my opinion. When you do "grep" you want exactly what you see in the logs. Here we concatenate value which is not visible in the code, so possibly someone in the future will waste time on debugging due to that.

Reply:
As I mentioned earlier, I require the click event to be triggered. Other generated events, such as untrusted events, are not useful for my use case.

The dynamic identifiers provided by the Leaflet-style code are effective for firing multiple generic event types (click, doubleclick, tripleclick). Setting a breakpoint in the event handler makes it easy to debug and inspect the call stack.

Reply:
ok let's get this one solved and tested